docs/team/juliapoo.md

```diff
@@ -0,0 +1,11 @@
+---
+  layout: default.md
+  title: "Jule's Project Portfolio Page"
+---
+
+### Project: CCA Manager
+
+CCA Manager is a **desktop app for managing contacts, optimized for use via a  Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, CCA Manager can get your contact management tasks done faster than traditional GUI apps.
```

Comment:
Just checking, do the files need to be capitalised according to our GH usernames, or is all lower case fine? Or maybe it doesn't matter at all?

Reply:
Nothing is stated about the portfolio files, but the profile pictures are required to be lower cased versions of our GH usernames.